tests/e2e/local/minikube/install_prereqs_macos.sh

```diff
@@ -1,5 +1,6 @@
 #!/bin/bash
 
+# shellcheck source=tests/e2e/local/common_macos.sh
```

Comment:
Can you please add double quotation marks like in the other scripts or remove from other to keep it consistent?

Reply:
you mean make it this way?

```
SCRIPTPATH="$(cd "$(dirname "$0")" ; pwd -P)"
ROOTDIR="$(dirname "${SCRIPTPATH}")"
# shellcheck source=tests/e2e/local/common_macos.sh
source "${ROOTDIR}/common_macos.sh"
```

Reply:
Take a look at `tests/e2e/local/minikube/install_prereqs_debian.sh`.
Over there it's `source "../common_linux.sh"` and here `source ../common_macos.sh`. Just make it consistent.

Reply:
I agree with @lichuqiang - we should not make too much changes in this PR for the sake of merging this fix ASAP.

Regarding to separate or merge these files, we can discuss it later - I separated them for better readability. Make sense?

Reply:
> feel it may be better to just combine the first two lines since SCRIPTPATH is only referenced in the second line anyway.

Hmm, I thought about it too. What I worried about it is that would make the statement too complex to understand. So, I'd prefer to leave them  independent to make it clearer.
And seems scripts that in need of parent path(e.g bin/ci2gubernator.sh) also behaviors the same way.
WDYT?